lib/compiler.js

```diff
@@ -35,7 +36,15 @@ function parseErrors(dirPath, errors) {
 function findImports(dirPath) {
   return function(file) {
     try {
-      return {contents: fs.readFileSync(path.resolve(dirPath, file), 'utf8')}
+      var filePath = path.resolve(dirPath, file);
+      // If file doesn't exist try in project node modules
+      if(!fs.existsSync(filePath)) {
+        filePath = path.resolve(
+          findNodeModules({cwd: dirPath, relative: false})[0],
```

Comment:
I would offer to use [`fs#readDirSync`](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options) instead

Reply:
Yeah sure completely forgot about that.